### PR TITLE
javascript-timers

### PR DESCRIPTION
**Pull request by easton-morris**
setInt
setInt2

### CODE REVIEW

File: javascript-timers/interval.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  } else if (header.textContent === '3') {
+    header.textContent = '2';
+  } else if (header.textContent === '2') {
+    header.textContent = '1';
+  } else if (header.textContent === '1') {
+    header.textContent = '~Earth Beeeelooowww Us~';
+  } else if (header.textContent === '~Earth Beeeelooowww Us~') {
+    window.clearInterval(intID);
+  }
+}, 2.0 * 1000);
```
Comment (TimelikeClosure) [inline javascript-timers/interval.js:14]: Do you normally count down at half speed?

File: javascript-timers/interval.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+    header.textContent = '3';
+  } else if (header.textContent === '3') {
+    header.textContent = '2';
+  } else if (header.textContent === '2') {
+    header.textContent = '1';
+  } else if (header.textContent === '1') {
+    header.textContent = '~Earth Beeeelooowww Us~';
+  } else if (header.textContent === '~Earth Beeeelooowww Us~') {
+    window.clearInterval(intID);
+  }
```
Comment (TimelikeClosure) [inline javascript-timers/interval.js:13]: There's probably a less brute-force way to do this.

**Review by TimelikeClosure — CHANGES_REQUESTED** [review]

File: javascript-timers/interval.js (hunk 1 above)
Comment (easton-morris) [reply javascript-timers/interval.js:14]: So what if I do? LOL. but i'll change it. it's more for mental pacing

File: javascript-timers/interval.js (hunk 2 above)
Comment (easton-morris) [reply javascript-timers/interval.js:13]: setInt3

**Review by TimelikeClosure — APPROVED** [review]

**Pull request merged**